src/Core/Utils/Color.hpp

```diff
@@ -10,12 +10,20 @@ namespace Ra {
 namespace Core {
 namespace Utils {
 
+/**
+ * Get a named color from its name. Return the uint value of named svg color. The value is of the
+ * form 0xRRGGBB. RR is red, GG is green and BB is blue. \param name: the name of the color as given
+ * from svg spec. \return Color code, or 0 if \a name is not a valid svg color name (0 is
+ * corresponds to black). \see https://www.december.com/html/spec/colorsvg.html
+ */
```

Comment:
Do we need this function to be public ?
The private static function of the Color class seems to be enough to me.

Reply:
Well the private static function need to use this one. My idea here is to have the map as a static in cpp instead of having it in .inl
I'm not sure how to do without getting the name to color map in headers. Suggestion welcome

Reply:
I don't understand: I see the conversation has been marked resolved, however the public function is still there.
Did you try @MathiasPaulin's proposal ?